inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/FlinkService.java

```diff
@@ -285,6 +285,8 @@ private String[] genProgramArgsV2(FlinkInfo flinkInfo, FlinkConfig flinkConfig)
         list.add(flinkInfo.getLocalConfPath());
         list.add("-checkpoint.interval");
         list.add("60000");
+        list.add("-metrics.audit.proxy.hosts");
+        list.add(flinkConfig.getAuditProxyHosts());
         return list.toArray(new String[0]);
```

Comment:
please remove the `genProgramArgs` method and rename `genProgramArgsV2` to `genProgramArgs`, I think it will not be used again.